org.osgi.test.junit5/src/main/java/org/osgi/test/junit5/context/InstalledBundleExtension.java

```diff
@@ -98,7 +106,13 @@ static Store getStore(ExtensionContext extensionContext) {
 	protected Object fieldValue(Field field, ExtensionContext extensionContext) {
 		assertFieldIsOfType(field, Bundle.class, supported, ExtensionConfigurationException::new);
 		InjectInstalledBundle injectBundle = field.getAnnotation(supported);
-		return installedBundleOf(injectBundle, extensionContext);
+		try {
+			return installedBundleOf(injectBundle, extensionContext);
+		} catch (Exception e) {
+			throw new ExtensionConfigurationException(String
+				.format("@InjectInstalledBundle [%s]: couldn't resolve bundle [%s]: %s", field.getName(),
```

Comment:
```suggestion
				.format("@%s [%s]: couldn't resolve bundle [%s]: %s", supported.getSimpleName(), field.getName(),
```

Don't hard code annotation name and use `supported` instead.